<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conversions</class>
 <widget class="QWidget" name="Conversions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>666</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="rootLayout">
   <item>
    <widget class="QLabel" name="Conversions_lbl">
     <property name="text">
      <string>Conversions</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="typeLayout">
     <item>
      <widget class="QLabel" name="Conversions_type_lbl">
       <property name="text">
        <string>Conversions type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Length</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>weight</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="inputLayout">
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Result">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Result</string>
     </property>
     <layout class="QVBoxLayout" name="resultLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_1">
          <property name="text">
           <string>Result 1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_1"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Result 2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Result 3:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Result 4:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
